packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -1095,6 +1121,11 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		if (this.selection && this.selection.isSelected(row)) {
 			classes.push("selected");
 		}
+
+		if (this.rowExpandedFn(row)) {
+			classes.push("row-collapsed");
```

Comment:
No direct styling is applied by default but it could be used later by the user the adapt the theming, otherwise the users won't have a way to differentiate between a regular and collapsed row.
I agree to rename it to `row-expanded` or something else but I don't see any reference to the simular classes you mention?

Reply:
In the CSS of the table, you added:
```css
tr.collapsed-row {
    height: 0;

    .mat-column-collapseDetail {
      padding: 0;
      border: 0;
    }
...
```

Or maybe we could simply add a class "expanded" in case the row is expanded ?